README.md

```diff
@@ -48,3 +48,12 @@ At first startup Eclipse IDE will checkout the code, perform a full build and co
 For further information, building and deployment instructions, please visit the [getting started page](https://wiki.eclipse.org/Kura/Getting_Started).
 
 To raise an issue, please report a bug on [GitHub issues](https://github.com/eclipse/kura/issues/new).
+
+## Supported Development Platforms
+The Eclipse Installer based setup works for the main used platforms like Linux, Mac Os and Windows.
+
+
+## Known issues
+Currently, the emulator web ui is not properly working on Windows so, with your setup, you will be able to build and deploy you applications, but not be able to use the Eclipse IDE based Kura emulator.
+
+The full build of Kura is only supported for Linux and Mac Os based systems.
```

Comment:
@cdealti The initial oomph build skips tests in any case. But they worked the last time I checked, so that note should not be necessary.